crates/test-utils/src/deploy_helpers.rs

```diff
@@ -674,122 +678,34 @@ pub struct ContractDeploymentArgs<T: Tokenize> {
     pub constructor_args: T,
 }
 
-pub enum TestContext {
-    Simple,
-    Counter,
-    PlainOpcodes,
-    ERC20,
+pub struct KakarotTestEnvironmentContext {
```

Comment:
Should we do a PR and a refactor workshop for this struct?
As @ClementWalter mentioned, it feels like there is some redundancy

Reply:
Yeah created a pr here https://github.com/kkrt-labs/kakarot-rpc/issues/560, let me know what I can add to the issue